app/views/declaration/office_of_exit_supplementary.scala.html

```diff
@@ -49,7 +53,9 @@
                   messages("standard.officeOfExit.empty"),
                   AutoCompleteItem.fromOfficeOfExit(OfficeOfExits.all)
               )
-
+        @components.details(messages("site.details.summary_text_these")){
```

Comment:
Probably I missed this last time, but we're not using `_` in the messages keys.

I'm not saying this is wrong, but it's just probably different convention.

Can we change `summary_text_these` to `summaryTextThese`?

Additionally what this `these` mean in this message key?

Reply:
I used _ convention beacause already we are using this in lot of places in messages.en
for example :
site.save_and_come_back_later = Save and come back later

we have two different summary link message
site.details.summary_text_this = For more information about this
site.details.summary_text_these = For more information about these
one of this and other for these

Reply:
I understand, this convention `site.save_and_come_back_later = Save and come back later` is probably autogenerated by old library, but it's okay